### PR TITLE
[Fix](mlu-ops): Fix an error of perf analyser

### PR DESCRIPTION
**Pull request by ZhouxiangZhouxiang**
Fix the name of prototype so that there is no duplicated names.

Thanks for your contribution and we appreciate it a lot. :rocket::rocket:

## 1. Motivation
When we run the perf tool to compare multiple operators, the comparison can fail. The reason is that 2 dataframes need to be merged according to their common keys called "protoName". ProtoNames are the names of calculators' testcases, which means there are a lot of repetitive ones like case0 or case_0. But as a key, protoName should be unique in each dataframe. Repetitive names means that merging 2 dataframes can result in incorrect results.
## 2. Modification
I have changed the name of the protoname in function preprocess in file parser.py to the full path of the test case of each calculator so that there are no repeated protoNames.
## 3. Test Report
This is the png generated by the python file for all the calculators, which means the now the bug is solved.
next

### CODE REVIEW

File: tools/perf_analyse/processor.py [hunk 1]
```diff
@@ -1216,4 +1216,4 @@ def run(self, dfs):
             print("the ouput comparison information excel is " +
                   compare_xlsx_path)
         logging.info("Processor run end")
-        return 0
+        return 0
```
Comment (DanieeelLiu) [inline tools/perf_analyse/processor.py:1219]: 这个文件没有代码修改就不用提交了

File: tools/perf_analyse/parser.py [hunk 2]
```diff
@@ -61,7 +61,9 @@ def is_testsuite_for_mluop_op(self, test_name):
 
     def preprocess(self, df):
         # protoName and mlu_platform are used to merge database
-        df['protoName'] = df['file_path'].apply(lambda x: x.split("/")[-1])
+        df['protoName'] = df['file_path']##.apply(lambda x: x.split("/")[-1])
+        ##print(df['protoName'])
```
Comment (PetrelYy) [inline tools/perf_analyse/parser.py:65]: 调试代码请勿合入

**Comment by chqy99** [comment]
Motivation 需要详细介绍为什么这样改的原因，而不是错误现象：多个算子有可能有同名的测例文件，使用测例的文件名当做键值不合理，应该使用测例的绝对路径当做键值

File: tools/perf_analyse/processor.py (hunk 1 above)
Comment (ZhouxiangZhouxiang) [reply tools/perf_analyse/processor.py]: 已修改。

File: tools/perf_analyse/parser.py (hunk 2 above)
Comment (ZhouxiangZhouxiang) [reply tools/perf_analyse/parser.py]: 已修改。

**Review by DanieeelLiu — APPROVED** [review]

**Review by chqy99 — APPROVED** [review]

**Review by PetrelYy — APPROVED** [review]

**Pull request merged**